pkg/scannerv4/mappers/mappers_test.go

```diff
@@ -1247,6 +1249,30 @@ func Test_toProtoV4VulnerabilitiesMap(t *testing.T) {
 				},
 			},
 		},
+		"when issued time is empty, use NVD published time": {
+			ccVulnerabilities: map[string]*claircore.Vulnerability{
+				"foo": {
+					ID:      "foo",
+					Name:    "CVE-2021-44228",
+					Updater: "unknown updater",
+				},
+			},
+			nvdVulns: map[string]map[string]*nvdschema.CVEAPIJSON20CVEItem{
+				"foo": {
+					"CVE-2021-44228": {
+						ID: "CVE-2021-44228",
+						Published: "2021-12-10T10:15:09.143",
```

Comment:
super duper nit: Noticed this is the same 'string' used to create `proto2021`.

Perhaps change definition of `proto2021` to `proto2021Str` and use it here, and then in the next element use `protoconv.ConvertTimeString(proto2021Str)` inline? /shrug